applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -260,17 +244,16 @@ template <unsigned int TDim, unsigned int TNumNodes>
 void UPwBaseElement<TDim, TNumNodes>::CalculateLeftHandSide(MatrixType&        rLeftHandSideMatrix,
                                                             const ProcessInfo& rCurrentProcessInfo)
 {
-    KRATOS_TRY;
-
-    // Calculation flags
-    const bool CalculateStiffnessMatrixFlag = true;
-    const bool CalculateResidualVectorFlag  = false;
-    VectorType TempVector;
+    KRATOS_TRY
 
-    CalculateAll(rLeftHandSideMatrix, TempVector, rCurrentProcessInfo, CalculateStiffnessMatrixFlag,
-                 CalculateResidualVectorFlag);
+    rLeftHandSideMatrix = ZeroMatrix{this->GetNumberOfDOF(), this->GetNumberOfDOF()};
```

Comment:
I'm not sure. Since this was the only place where we didn't set the left hand side matrix to a zero matrix of the correct sizes, I thought it would be better to be consistent and at the safe side.